src/views/httpResponseWebview.ts

```diff
@@ -45,12 +45,13 @@ export class HttpResponseWebview extends BaseWebview {
         this.context.subscriptions.push(commands.registerCommand('rest-client.unfold-response', () => this.unfoldResponseBody()));
     }
 
-    public async render(response: HttpResponse, column: ViewColumn) {
+    public async render(response: HttpResponse, request: HttpRequest, column: ViewColumn) {
```

Comment:
There are two kinds of views of previewing responses, one is the `webview` here, the other is the `untitled document view`, does this setting applies in both views?

Reply:
Doesn't apply for `untitled document view`, updated the README accordingly.